README.md

```diff
@@ -297,12 +297,14 @@ You can also substitute the backend with `torch_stat` to run a reference compari
 # Add a model test
 If you want to record run time metrics for a model or test, include a Pytest fixture named `record_property` as a parameter and set the "model_name" key.  
 If you also want to compile the model with torch_ttnn backend, set the "torch_ttnn" key to a tuple in this order `(model, test_inputs, outputs)`. "model_name" still needs to be set. See the example code snippet below. Currently, only `torch.nn.Module` models with a `forward` function are supported.
-```
+```python
 def Model(torch.nn.Module):
     def forward(self, x):
         # ...
         return outputs
 
+# Add torch_only marker if torch/CPU runs, but compiled version is xfail
+@pytest.mark.torch_only
```

Comment:
@kevinwuTT lets find a better name.

This should indicate that we know that this fails run on tenstorrent hardware and we are okay with it for now.


Maybe `compile_fail_expected`?

Reply:
@ayerofieiev-tt `compilation_xfail` to correspond to the native `pytest.xfail` maybe?

Reply:
@kevinwuTT , sounds good